test/quantization/core/test_quantized_op.py

```diff
@@ -253,7 +255,12 @@ def test_qrelu(self, X):
                 }
             }
         ]
-        self._test_activation_function(X, 'relu', relu_test_configs)
+        devices = ["cpu", "cuda"] if TEST_CUDA else ["cpu"]
+        for device in devices:
+            X_tensor = list(copy.deepcopy(X))
+            X_tensor[0] = torch.from_numpy(X_tensor[0])
+            X_tensor[0] = X_tensor[0].to(device)
```

Comment:
I think these lines shouldn't be here, we initialize torch.Tensor from numpy ndarray in L170, we should do this there as well

Reply:
also why do we get X_tensor[0]? is this just for testing?

Reply:
I changed this part and make it consistent with test_qgelu().